perception/autoware_traffic_light_selector/src/traffic_light_selector_node.cpp

```diff
@@ -160,8 +144,28 @@ void TrafficLightSelectorNode::objectsCallback(
     debug_publisher_->publish<autoware_internal_debug_msgs::msg::Float64Stamped>(
       "debug/pipeline_latency_ms", pipeline_latency_ms);
   }
-  return;
 }
+
+void TrafficLightSelectorNode::evaluateWholeRois(
+  const std::vector<RegionOfInterest> & detected_rois,
+  const std::map<uint8_t, RegionOfInterest> & expect_rois_shifted_map, double & total_max_iou,
+  std::map<uint8_t, RegionOfInterest> & total_max_iou_rois_map)
```

Comment:
Plz recheck type as the same above comment
```suggestion
void TrafficLightSelectorNode::evaluateWholeRois(
  const std::vector<RegionOfInterest> & detected_rois,
  const std::map<int64_t, RegionOfInterest> & expect_rois_shifted_map, double & total_max_iou,
  std::map<int64_t, RegionOfInterest> & total_max_iou_rois_map)
```

Reply:
It is true. Fixed in e4418b4